package.json

```diff
@@ -80,7 +80,7 @@
     "@material-ui/core": "^4.11.0",
     "@metamask/contract-metadata": "^1.22.0",
     "@metamask/controllers": "^5.1.0",
-    "@metamask/eth-ledger-bridge-keyring": "^0.2.6",
+    "@metamask/eth-ledger-bridge-keyring": "darkwing/eth-ledger-bridge-keyring#ledger-live",
```

Comment:
This will obviously need to be changed once we publish changes on `metamask/eth-ledger-bridge-keyring`